tests/core/core.spec.ts

```diff
@@ -887,6 +887,37 @@ suite("@dataform/core", () => {
         });
       }
     );
+
+    test("schema without suffix", () => {
```

Comment:
generally for suffix/suffix-less tests we do some table-driven stuff, i.e. something like

```
for ({config, expectedTarget of [{config: TestConfigs.redshift, expectedTarget: "schema.test"}, ...]) {
  test("schema with config " + JSON.stringify(config), () => {
    // test
  }
}
```

Reply:
Thanks, updated